packages/web-components/src/components/va-language-toggle/va-language-toggle.tsx

```diff
@@ -0,0 +1,102 @@
+import { Component, Host, h, State, Event, EventEmitter, Prop, Fragment } from '@stencil/core';
+import classNames from 'classnames';
+
+export type LangUrl = {
+  label: string;
+  href: string;
+  lang: string;
+};
+
+/**
+ * @componentName Language Toggle
+ * @maturityCategory caution
+ * @maturityLevel candidate
+ */
+
+@Component({
+  tag: 'va-language-toggle',
+  styleUrl: 'va-language-toggle.scss',
+  shadow: true,
+})
+export class VaLanguageToggle {
+  /**
+   * The ISO language code for the page. Default is 'en'.
+   */
+  @Prop() language: string = 'en';
+
+  /**
+   * A JSON array of objects with link data. Each object should have an href, lang (ISO language code), and label properties. If using the pure web component provide as a string. Example: `[{"href": "/one", "lang": "en", "label": "English"}, ...]`.
+   */
+  @Prop() urls: LangUrl[] | string;
```

Comment:
Our guidelines say that we only support 3 languages; English, Espanol, Tagalog

I don't know if we want to add this type of flexibility.

> Other language options. VA.gov supports the 3 languages identified in the toggle. Do not add or remove from those 3 languages.

Reply:
@jamigibbs - good point - I updated the component to only allow en, es, or tl.